incubator/podgrab/1.0.0/questions.yaml

```diff
@@ -0,0 +1,505 @@
+                label: "Mount a ramdisk instead of actual storage"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+              - variable: accessMode
+                label: "Access Mode (Advanced)"
+                description: "Allow or disallow multiple PVC's writhing to the same PVC"
+                schema:
+                  type: string
+                  default: "ReadWriteOnce"
+                  enum:
+                    - value: "ReadWriteOnce"
+                      description: "ReadWriteOnce"
+                    - value: "ReadOnlyMany"
+                      description: "ReadOnlyMany"
+                    - value: "ReadWriteMany"
+                      description: "ReadWriteMany"
+              - variable: size
+                label: "Size quotum of storage"
+                schema:
+                  type: string
+                  default: "1Gi"
+        - variable: data
+          label: "Asset Storage"
+          description: "Stores the downloaded podcasts."
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable the storage"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: true
+              - variable: storageClass
+                label: "Type of Storage"
+                description: " Warning: Anything other than Internal will break rollback!"
+                schema:
+                  type: string
+                  default: ""
+                  enum:
+                    - value: ""
+                      description: "Internal"
+              - variable: mountPath
+                label: "mountPath"
+                description: "Path inside the container the storage is mounted"
+                schema:
+                  type: string
+                  default: "/assets"
+                  hidden: true
+              - variable: emptyDir
+                label: "Mount a ramdisk instead of actual storage"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+              - variable: accessMode
+                label: "Access Mode (Advanced)"
+                description: "Allow or disallow multiple PVC's writhing to the same PVC"
+                schema:
+                  type: string
+                  default: "ReadWriteOnce"
+                  enum:
+                    - value: "ReadWriteOnce"
+                      description: "ReadWriteOnce"
+                    - value: "ReadOnlyMany"
+                      description: "ReadOnlyMany"
+                    - value: "ReadWriteMany"
+                      description: "ReadWriteMany"
+              - variable: size
+                label: "Size quotum of storage"
+                schema:
+                  type: string
+                  default: "1Gi"
```

Comment:
Persistance is just for storage that isn't likely to be accessed by the user (such as config files).
Downloads are stored in customStorage for all container, so I suggest doing the same here.

Reply:
I went back and forth on this since the downloads can be accessed/downloaded from the web interface of the app but none the less it's any easy change to make and some users may want to access the downloads from the filesystem level. :-)

Reply:
If you're in doubt ask yourself:
"If I revert a update, does this folder need to be reverted with it"
If the answer is "no", then most likely you want it as customStorage ;-)

Reply:
Fantastic! Great way to look at it :-)